crates/proof/src/eigenda_provider.rs

```diff
@@ -100,4 +104,75 @@ impl<T: CommsClient + Sync + Send> EigenDABlobProvider for OracleEigenDAProvider
 
         Ok(blob.into())
     }
+
+    /// get_blob_v2 takes a v2 cert type as opposed to bytes stream
+    async fn get_blob_v2(&mut self, cert: &EigenDAV2Cert) -> Result<Blob, Self::Error> {
```

Comment:
add documentation for how cert is converted to blob_key and what blob_key's encoding is.